packages/tools/fluid-runner/package.json

```diff
@@ -118,7 +118,6 @@
 		"temp-directory": "nyc/.nyc_output"
 	},
 	"dependencies": {
-		"@fluidframework/aqueduct": "workspace:~",
```

Comment:
Was only used for single use of `BaseContainerRuntimeFactory` in test code. Removed.